### ISSUE TITLE
Harvester and Cropsticks (Agricraft)

### ISSUE THREAD

**Issue by KEJWII** [PrinceOfAmber/Cyclic#2326]
Minecraft Version: 1.12.2

Forge Version: 14.23.5.2859

Mod Version: 1.20.12

Single Player or Server: Singleplayer

I guess Harvester doesn't collect crops in Cropsticks (Agricraft 2.12.0-1.12.2-b2)

**Comment by Lothrazar** [comment]
yeah there are certain mods that block automated growth sunch as bonemeal or watering cans or growth machines such as sprinklers. check their mods for config files to mabye unlock this feature if they have it. otherwise thats the way it works i guess.    we are keeping 1.12.2 stable outside of emergency fixes, so this will be treated working as intended in that version. as always new issue reports are welcome
